utils/splunk/splunk_helper.py

```diff
@@ -117,6 +117,8 @@ def _do_post(self, path, payload, request_timeout_seconds):
         try:
             resp.raise_for_status()
         except HTTPError as error:
-            self.log.error("Received error response with status {} and body {}".format(resp.status_code, resp.content))
-            raise error
+            self.log.warn("Received response with status {} and body {}".format(resp.status_code, resp.content))
```

Comment:
should be a `log.error` if `splunk_ignore_config` false

Reply:
ok, I will remove the warning and simply put the error in if `splunk_ignore_config` false block.